src/components/reorderer/js/Reorderer.js

```diff
@@ -595,7 +591,7 @@ https://github.com/fluid-project/infusion/raw/main/Infusion-LICENSE.txt
                 var handlePos = fluid.dom.computeAbsolutePosition(handle);
                 var handleWidth = handle.offsetWidth;
                 var handleHeight = handle.offsetHeight;
-                item.focus();
+                item.trigger("focus");
```

Comment:
This is an interesting corner where start and stop have become more asymmetric. I imagine it might create problems if we made this function async like stop, but it might be worth trying to see what happens. At the very least, we could convert this to a fluid.focus() call to make it consistent with stop, even if something goes wrong if we try to await() it

Reply:
Doing a straight conversation to `fluid.focus(item)` seems to be okay. Making `start` async and using `await` breaks the interaction. I'll just convert and not use async/await here.

Reply:
Actually it might be working, I accidentally left in a call to trigger("focus"). Will explore some more.

Reply:
Okay there is a small breakage when using async/await. Using the layout reordered demo if you move some items and then click on some empty space, you can end up not being able to position some of the reorder able elements. This doesn't seem to happen with just `fluid.focus` and no async/await.

Reply:
Thanks - could you add a code comment before the method reporting that it breaks if made async

Reply:
@amb26 I've added a comment. This is ready for more review.